contrib/zsh-completion/_packer

```diff
@@ -19,14 +24,39 @@ _packer () {
     '-only=[(foo,bar,baz) Only build the given builds by name.]'
     '-parallel=[(false) Disable parallelization. (Default: false)]'
     '-parallel-builds=[(0) Number of builds to run in parallel. (Defaults to infinite: 0)]'
+    '-timestamp-ui[Enable prefixing of each ui output with an RFC3339 timestamp]'
     '-var[("key=value") Variable for templates, can be used multiple times.]'
     '-var-file=[(path) JSON or HCL2 file containing user variables.]'
-    '(-)*:files:_files -g "*.json"'
+    '-warn-on-undeclared-var[Display warnings for user variable files containing undeclared variables.]'
+    '(-)*:files:_files -g "*pkr.{hcl,json}"'
+  )
+
+  local -a console_arguments && console_arguments=(
+    '-var[("key=value") Variable for templates, can be used multiple times.]'
+    '-var-file=[(path) JSON or HCL2 file containing user variables.]'
+    '(-)*:files:_files -g "*pkr.{hcl,json}"'
+  )
+
+  local -a fix_arguments && fix_arguments=(
+    '(-)*:files:_files -g "*pkr.{hcl,json}"'
```

Comment:
My only reservation here is that the fix command is not implemented for HCL2 templates.  Maybe we should keep to json files for now. What do you think?
```suggestion
    '(-)*:files:_files -g "*.json"'
```

Reply:
Oh, wasn't aware of that and your are absolutely right. It would be better to not complete hcl-based templates for the fix command. Now, that you already merged... shall we forget that or file a new issue? I'd be willing to fix that as it would be technically correct.

Reply:
@mrolli this is on me. I'll take care of the follow up PR. Thanks again for this contribution .

Reply:
I opened https://github.com/hashicorp/packer/pull/12366